### PR TITLE
feat: add react-specific env var banners, grey out private form rows

### PR DESCRIPTION
**Pull request by justynoh**
## Problem
As discussed.

Also allow markdown in banners

## Solution

**Breaking Changes** 
- No - this PR is backwards compatible  

## Deploy Notes

**New environment variables**:

- `IS_GENERAL_MAINTENANCE_REACT`
- `IS_LOGIN_BANNER_REACT`
- `SITE_BANNER_CONTENT_REACT`
- `ADMIN_BANNER_CONTENT_REACT`

### CODE REVIEW

File: frontend/src/features/admin-form/common/AdminFormLayout.tsx [hunk 1]
```diff
@@ -25,12 +25,14 @@ export const AdminFormLayout = (): JSX.Element => {
   const { formId } = useParams()
   if (!formId) throw new Error('No formId provided')
 
-  const { data: { siteBannerContent, adminBannerContent } = {} } = useEnv()
+  // TODO: Revert back to non-react banners post-migration.
```
Comment (karrui) [inline frontend/src/features/admin-form/common/AdminFormLayout.tsx]: create issue and add the issue number here? maybe can just link it under https://github.com/opengovsg/FormSG/issues/4279

File: frontend/src/features/workspace/WorkspacePage.tsx [hunk 2]
```diff
@@ -39,12 +39,14 @@ const useWorkspaceForms = () => {
 }
 
 export const WorkspacePage = (): JSX.Element => {
-  const { data: { siteBannerContent, adminBannerContent } = {} } = useEnv()
+  const { data: { siteBannerContentReact, adminBannerContentReact } = {} } =
+    useEnv()
 
+  // TODO: Revert back to non-react banners post-migration.
```
Comment (karrui) [inline frontend/src/features/workspace/WorkspacePage.tsx]: ^

File: src/app/config/config.ts [hunk 3]
```diff
@@ -223,6 +223,11 @@ const config: Config = {
   isLoginBanner: basicVars.banner.isLoginBanner,
   siteBannerContent: basicVars.banner.siteBannerContent,
   adminBannerContent: basicVars.banner.adminBannerContent,
+  // TODO: Delete these when react migration is over. Revert back to original banner variables in react frontend.
```
Comment (karrui) [inline src/app/config/config.ts]: ^

File: src/app/config/schema.ts [hunk 4]
```diff
@@ -181,6 +181,31 @@ export const optionalVarsSchema: Schema<IOptionalVarsSchema> = {
       default: '',
       env: 'ADMIN_BANNER_CONTENT',
     },
+    // TODO: Delete these when react migration is over. Revert back to original banner variables in react frontend.
```
Comment (karrui) [inline src/app/config/schema.ts]: ^

**Review by karrui — APPROVED** [review]
lgtm

**Review by timotheeg — APPROVED** [review]
LGTM

**Review by wanlingt — APPROVED** [review]

**Pull request merged**